src/stylesheets/abstracts/_variables.scss

```diff
@@ -17,18 +17,23 @@ $color-dark-purple: #252224; //text
 
 /*--------- Fonts ---------*/
 
+
 @font-face {
   font-family: 'Geomanist';
-  src: url(../src/fonts/geomanist/Geomanist-Regular.otf);
+  font-weight: 400;
```

Comment:
you can just set this to `font-weight: normal`. Sure, that's the same as a `400` font-weight, but I think `normal` makes the intention clearer.